terraform_files/variables-libvirt.tf

```diff
@@ -20,12 +20,32 @@ variable "cluster_domain" {
 
 variable "machine_cidr" {
   type        = string
-  description = "Cluster domain"
+  description = "Cluster IPv4 domain"
+}
+
+variable machine_cidr_addresses {
+    type = list(string)
+    description = "Addresses for machine CIDR network"
 }
 
 variable "provisioning_cidr" {
   type        = string
-  description = "Provisioning domain"
+  description = "Provisioning IPv4 domain"
+}
+
+variable provisioning_cidr_addresses {
+    type = list(string)
+    description = "Addresses for provisioning CIDR network"
+}
+
+variable "machine_cidr6" {
```

Comment:
I don't see any usage of this one?

Reply:
https://github.com/empovit/assisted-test-infra/blob/b54bd3829f6c4e83e6f1cacee481b81b0fe338d7/discovery-infra%2Fstart_discovery.py#L303

Reply:
i mean in terraform main.tf? if we don't use it there why we are providing data to it? Not sure for what we have it?